ECOSYSTEM.md

```diff
@@ -49,8 +49,10 @@ If your project isn't listed here and you would like it to be, please feel free
 - [axum-tutorial] ([website][axum-tutorial-website]): Axum web framework tutorial for beginners.
 - [demo-rust-axum]: Demo of Rust and axum web framework
 - [Introduction to axum (talk)]: Talk about axum from the Copenhagen Rust Meetup.
+- [Getting Started with Axum](https://carlosmv.hashnode.dev/getting-started-with-axum-rust): Axum tutorial, GET, POST endpoints and serving files.
```

Comment:
Markdown requires that you either put the link in parenthesis or in the bottom as a reference but not both.

```suggestion
- [Getting Started with Axum]: Axum tutorial, GET, POST endpoints and serving files.
```

Reply:
I didn't know that, thank you.